src/routers/DeleteUserRouter.ts

```diff
@@ -12,8 +12,7 @@ class DeleteUserRouter extends ApplicationRouter<void> {
   }
 
   async content(req: Request): Promise<void> {
```

Comment:
Do we need a `DeleteUserRouter`?

Reply:
It might be useful later if people want to remove their accounts.